doc/conf.py

```diff
@@ -49,40 +55,47 @@
 epub_scheme = 'url'
 epub_identifier = epub_publisher
 epub_pre_files = [('index.xhtml', 'Welcome')]
-epub_post_files = [('usage/installation.xhtml', 'Installing Sphinx'),
-                   ('develop.xhtml', 'Sphinx development')]
-epub_exclude_files = ['_static/opensearch.xml', '_static/doctools.js',
-                      '_static/searchtools.js',
-                      '_static/sphinx_highlight.js',
-                      '_static/basic.css',
-                      '_static/language_data.js',
-                      'search.html', '_static/websupport.js']
+epub_post_files = [
+    ('usage/installation.xhtml', 'Installing Sphinx'),
+    ('develop.xhtml', 'Sphinx development'),
+]
+epub_exclude_files = [
+    '_static/opensearch.xml',
+    '_static/doctools.js',
+    '_static/searchtools.js',
+    '_static/sphinx_highlight.js',
+    '_static/basic.css',
+    '_static/language_data.js',
+    'search.html',
+    '_static/websupport.js',
+]
 epub_fix_images = False
 epub_max_image_width = 0
 epub_show_urls = 'inline'
 epub_use_index = False
 epub_description = 'Sphinx documentation generator system manual'
 
-latex_documents = [('index', 'sphinx.tex', 'Sphinx Documentation',
-                    'the Sphinx developers', 'manual', 1)]
+latex_documents = [
+    ('index', 'sphinx.tex', 'Sphinx Documentation', 'the Sphinx developers', 'manual', 1)
+]
 latex_logo = '_static/sphinx.png'
 latex_elements = {
     'fontenc': r'\usepackage[LGR,X2,T1]{fontenc}',
-    'passoptionstopackages': r'''
+    'passoptionstopackages': r"""
 \PassOptionsToPackage{svgnames}{xcolor}
-''',
-    'preamble': r'''
+""",
+    'preamble': r"""
 \DeclareUnicodeCharacter{229E}{\ensuremath{\boxplus}}
 \setcounter{tocdepth}{3}%    depth of what main TOC shows (3=subsubsection)
 \setcounter{secnumdepth}{1}% depth of section numbering
 \setlength{\tymin}{2cm}%     avoid too cramped table columns
-''',
+""",
     # fix missing index entry due to RTD doing only once pdflatex after makeindex
-    'printindex': r'''
+    'printindex': r"""
```

Comment:
I assume that triple quotes will always be using double quotes and single quotes will be using... single quotes right?